ca_tools/package.xml

```diff
@@ -19,6 +19,7 @@
 
   <exec_depend>joy</exec_depend>
   <exec_depend>joy_teleop</exec_depend>
+  <exec_depend>draw_square</exec_depend>
```

Comment:
You set as dependency an executable that belongs to itself. You have to add here only external ROS packages that are being used. Did you understand?

Reply:
Not sure I'm understanding. If I dont include this line, when I run 'catkin_make' it prints an error:

```console
catkin_package() DEPENDS on the catkin package 'draw_square' which must
  therefore be listed as a run dependency in the package.xml
Call Stack (most recent call first):
  /opt/ros/kinetic/share/catkin/cmake/catkin_package.cmake:102 (_catkin_package)
  create_autonomy/ca_tools/CMakeLists.txt:6 (catkin_package)
```

Reply:
This message is because you also have an error [here](https://github.com/RoboticaUtnFrba/create_autonomy/pull/103/files#diff-e45551332c403f497bed4c72f36bb41fR11).

It's not necessary to declare Python scripts as dependencies. You have to set it as executable. Check [here](https://answers.ros.org/question/56640/calling-a-python-script-with-rosrun/).

Reply:
Ok, I understand now.